### DISCUSSION TITLE
Is there a way to setup a Bun script to use `tsconfig.bun.json` instead of `tsconfig.json`?

### DISCUSSION THREAD

**Discussion by nelson6e65** [oven-sh/bun#5725] [Q&A] [answered]
I have to have 3 different `tsconfig.*.json` files, and I like to use `tsconfig.bun.json` for my specific bun script, but have not found a way to customize the `tsconfig` file to use; it always uses tsconfig.json.

`script1.ts`

```ts
#!/usr/bin/env bun

// ...
```

For example, I don't want to use the main sources aliases in my scripts files.

**Comment by melroy89** [comment]
Bun will just use `tsconfig.json`.

**Comment by okineadev** [comment]
I would really need this too!

**Reply by RiskyMH** [reply, marked as answer]
You can use the `--tsconfig-override` flag that I fixed in #21045. If you don't have access to the raw cli, then you can add it to the BUN_OPTIONS environment variable.
